main.tf

```diff
@@ -3,6 +3,11 @@ locals {
     type  = split(".", k)[0]
     field = join(".", slice(split(".", k), 1, length(split(".", k))))
   }) if var.create_graphql_api }
+
+  functions = { for k, v in var.functions : k => merge(v, {
```

Comment:
Why do we need this for `functions`? I think it will work well without it.

Reply:
Makes sense. I've simplified it.